packages/assistify-help-request/assets/stylesheets/helpRequestContext.less

```diff
@@ -47,3 +47,29 @@
 		}
 	}
 }
+
+& .rc-input__textarea {
```

Comment:
@vickyokrm this should not be a class outside the assistify-control-elements. 
The way it's currently selected, it would match all `.rc-input__textarea` in the application, it should be active only on the creation tab